pytest.ini

```diff
@@ -19,6 +19,7 @@ env =
     AWS_REGION=ca-central-1
     ASSET_DOMAIN=assets.notification.canada.ca
     NOTIFY_EMAIL_DOMAIN=notification.canada.ca
+    FF_NOTIFICATION_CELERY_PERSISTANCE=1
```

Comment:
Yeah I agree. I thought about this but then thought we would be testing in staging and so we could potentially get away with it since we're planning to remove the old code anyways if all is well or revert if it is not